source/read/read-preference.txt

```diff
@@ -0,0 +1,131 @@
+.. _java-rs-read-preference:
+
+===============
+Read Preference
+===============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, preference, option
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+To choose which member of a replica set to read from, you can configure read
+preferences with the {+driver-short+}.
```

Comment:
S: to avoid "with"
```suggestion
To choose which member of a replica set to read from, you can use the {+driver-short+} to configure read
preferences.
```